spec/queries/users/get_user_spec.rb

```diff
@@ -1,47 +1,53 @@
 require 'rails_helper'
 module Queries
-  RSpec.describe Types::QueryType do
+  RSpec.describe Types::QueryType, type: :request do
     before :each do
-      User.create(id: '1', email: 'john@email.com', username: 'John', password: '1234')
-      User.create(id: '2', email: 'jared@email.com', username: 'Jared', password: '1234')
-      
-      @get_user = <<-GRAPHQL
-                                  query($id: ID!) {
-                                      getUser(id: $id) {
-                                          id
-                                          username
-                                          email
-                                      }
-                                  }
-      GRAPHQL
+      @user_1 = create(:user)
+      @user_2 = create(:user, :with_availabilities)
```

Comment:
You can use this syntax if you want to create a factorybot user that has 3 existing availabilities